madmin.gemspec

```diff
@@ -19,5 +19,5 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.5.0"
 
   spec.add_dependency "rails", ">= 6.0.3"
-  spec.add_dependency "pagy", ">= 3.5", "< 5.0"
+  spec.add_dependency "pagy", ">= 5.0", "< 6.0"
```

Comment:
Is there a reason we need to bump the minimum version? The lower we can allow it makes it more compatible with apps that use an older pagy.

Reply:
Right... I didn't take that into account.

Reply:
That's alright! Sometimes there are breaking changes so I was just wondering. 👍